client/SDL/CMakeLists.txt

```diff
@@ -29,31 +55,49 @@ if (WITH_DEBUG_SDL_KBD_EVENTS)
 	add_definitions(-DWITH_DEBUG_SDL_KBD_EVENTS)
 endif()
 
-include_directories(${SDL2_INCLUDE_DIRS})
+find_package(SDL2 REQUIRED)
+include_directories(${SDL2_INCLUDE_DIR})
+
+# Here we add dependencies for stand alone builds.
+# Only add these if not build within FreeRDP source tree
+if (CMAKE_SOURCE_DIR STREQUAL CMAKE_CURRENT_SOURCE_DIR)
+	find_package(WinPR REQUIRED)
+	include_directories(${WinPR_INCLUDE_DIR})
+
+	find_package(FreeRDP REQUIRED)
+	include_directories(${FreeRDP_INCLUDE_DIR})
+
+	find_package(FreeRDP-Client REQUIRED)
+	include_directories(${FreeRDP-Client_INCLUDE_DIR})
+endif()
+
 set(SRCS
-    sdl_utils.c
+    sdl_utils.cpp
     sdl_utils.h
```

Comment:
I think the header files could be adjusted too (`.h` -> `.hpp`).

Reply:
done